### ISSUE TITLE
TimeZone false positive on .new from another class

### ISSUE THREAD

**Issue by aripollak** [bbatsov/rubocop#1900]
This causes an incorrect failure in the TimeZone cop:
```ruby
C: Rails/TimeZone: Do not use DateTime.new without zone. Use DateTime.zone.local instead.
    Poop.new DateTime.iso8601(start_at)
                      ^^^^^^^
```

But this works fine:
```ruby
    Poop.hello DateTime.iso8601(start_at)
```

**Comment by palkan** [comment]
Use `acceptable` style to bypass such offenses.

**Comment by aripollak** [comment]
Was this closed because of the `acceptable` suggestion? If so, I don't think that's really an equivalent workaround.

**Comment by bbatsov** [comment]
What's wrong with using `acceptable`?

**Comment by aripollak** [comment]
Actually, the bug seems to be fixed in the latest version of Rubocop anyway. The problem had been  that it thought the `.new` was called on DateTime instead of the other class.
